src/index.ts

```diff
@@ -1,12 +1,50 @@
-import express, { Application, Request, Response } from 'express';
+import express, { Application, Request, Response, NextFunction} from 'express';
 
-let indexRouter = require('./routes/home');
+const dotenv = require('dotenv');
+dotenv.config()
+
+const createError = require('http-errors');
+const path = require('path');
+
+const indexRouter = require('./routes/main');
+const badgeclassRouter = require('./routes/badgeclass')
+const assertionRouter = require('./routes/assertion')
 
 const PORT = 5000;
 const app: Application = express();
 
 
 // routes setup
+app.use(express.json());
+app.use(express.urlencoded({ extended: true }));
+app.use(express.static(path.join(__dirname, 'public')));
+
 app.use('/', indexRouter);
+app.use('/badgeclass', badgeclassRouter)
+app.use('/assertion', assertionRouter)
 
+// database setup
+const mongoose = require('mongoose');
+mongoose.set('useFindAndModify', false);
+const mongoDB = "mongodb://" + process.env.DB_USER + ":" + process.env.DB_PASSWORD + "@ds231529.mlab.com:31529/wise-badges-dev"
```

Comment:
I'd store this entire string in the environment file instead of the user and password. Someone using their own DB will want to replace the whole string. If we end up dockerizing and running our own mongodb image, this will also be necessary.

You can also construct this with a template string

```suggestion
const mongoDB = `mongodb://${process.env.DB_USER}:${process.env.DB_PASSWORD}@ds231529.mlab.com:31529/wise-badges-dev`
```